### DISCUSSION TITLE
Adding cluster node to optimize bandwidth

### DISCUSSION THREAD

**Discussion by mausimus** [rabbitmq/rabbitmq-server#3812] [Questions] [answered]
Quick question about routing in a cluster:
* My RabbitMQ node and Producer are in the Data Center
* Multiple Clients are in the Office but all receive the same messages from the Producer (broadcast)

My aim is for messages to be sent from the Data Center to the Office only once and then distributed locally to Clients (massive bandwidth saving).

As I understand from the docs I will need to:
* set up an additional clustered node in the Office
* have Clients connect and declare their private queues on the Office node while binding to the exchange declared on the Data Center node

However is this enough for RabbitMQ to recognize they all bind to the same exchange and only send messages between nodes once, rather than separately for each Client? Or will I need to set up a 'local proxy' exchange on the Office node bound to Data Center's exchange to force such flow?

**Comment by lukebakken** [comment, marked as answer]
Do not add nodes to the cluster in your office. Use Federation or a Shovel to move messages from the Data Center to the Office.
https://www.rabbitmq.com/federation.html
https://www.rabbitmq.com/shovel.html
